pkg/controller/podautoscaler/horizontal.go

```diff
@@ -370,6 +273,134 @@ func (a *HorizontalController) reconcileKey(key string) error {
 	return a.reconcileAutoscaler(hpa)
 }
 
+// Computes the desired number of replicas for the specified metric of type ObjectMetricSourceType.
+func (a *HorizontalController) computeReplicasForObjectMetric(currentReplicas int32, metricSpec autoscalingv2.MetricSpec, hpa *autoscalingv2.HorizontalPodAutoscaler, selector labels.Selector, status *autoscalingv2.MetricStatus) (int32, time.Time, string, error) {
```

Comment:
name should imply that it's also setting status

Reply:
Done.